diff --git a/electron-react-boilerplate/src/main/main.ts b/electron-react-boilerplate/src/main/main.ts
index 327b81f..84cabb0 100644
--- a/electron-react-boilerplate/src/main/main.ts
+++ b/electron-react-boilerplate/src/main/main.ts
@@ -73,4 +73,5 @@ const createWindow = async () => {
     show: false,
     width: 1024,
+    minWidth: 330,
     height: 728,
     icon: getAssetPath('icon.png'),
@@ -82,5 +83,5 @@ const createWindow = async () => {
   });
 
-  mainWindow.loadURL(resolveHtmlPath('index.html'));
+  mainWindow.loadURL('http://localhost:19000');
 
   mainWindow.on('ready-to-show', () => {
